### PR TITLE
Modified so that vbf templates can be generated

### PR DESCRIPTION
**Pull request by andresnava1000**

### CODE REVIEW

File: src/HHbbVV/postprocessing/postprocessing.py [hunk 1]
```diff
@@ -392,6 +411,31 @@ def main(args):
...
+        for key, df in events_dict.items():
+            # for each JES,JER, and JMS JMR correction
+            ptlabel = "_JES_up"
+            mlabel = "_JMS_down"
+
+            bb_mask = (
+                df[("ak8FatJetParticleNetMD_Txbb", 0)] > df[("ak8FatJetParticleNetMD_Txbb", 1)]
+            )
+
+            if key == "Data":
```
Comment (rkansal47) [inline src/HHbbVV/postprocessing/postprocessing.py:427]: won't this mean you're not saving the nominal (non up/down) values for non-Data?

**Comment by rkansal47** [comment]
@andresnava1000 could you take a look at the comment ^, and then I can merge this.

File: src/HHbbVV/postprocessing/postprocessing.py (hunk 1 above)
Comment (andresnava1000) [reply src/HHbbVV/postprocessing/postprocessing.py:427]: Ya, oops I thought I had pushed an edit for this but the if key == "Data": part gets turned into  "_add_vbf_columns(df, bb_mask, ptlabel="", mlabel="")

            if key == "Data":
                continue"

Comment (andresnava1000) [reply src/HHbbVV/postprocessing/postprocessing.py:427]: I can add that it

**Comment by rkansal47** [comment]
ok @andresnava1000  sorry there's something crazy going on with this PR (1000+ commits??) - could you just make a new branch from main and copy over your changes to the `hh_vars.py` and `postprocessing.py` scripts?

**Comment by andresnava1000** [comment]
@rkansal47 I was getting some weird issues with git when trying to use rebase so I tried using "git rebase origin/main vbf_template" instead but then I got a lot of conflict issues. I made a new branch instead which has all of the same edits.

**Comment by rkansal47** [comment]
ok should I close this pull request without merging then?

**Pull request closed without merging**